packages/integrations/cloudflare/README.md

```diff
@@ -75,10 +75,23 @@ It's then possible to update the preview script in your `package.json` to `"prev
 
 You can access all the Cloudflare bindings and environment variables from Astro components and API routes through `Astro.locals`.
 
-```js
+If you're inside an `.astro` file, you access the runtime using the `Astro.locals` global:
+
+```astro
 const env = Astro.locals.runtime.env;
```

Comment:
What do you think about this change? In the sentence above you talked about `runtime`, so I think it makes sense to update the example, not sure if we should show that `env` is inside it.
```suggestion
const runtime = Astro.locals.runtime;
```